benchmarks/rayleigh_taylor_instability_free_surface/drip_depth.cc

```diff
@@ -0,0 +1,154 @@
+/*
+  Copyright (C) 2011 - 2017 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+#include "drip_depth.h"
+#include <aspect/geometry_model/box.h>
+#include <aspect/simulator_access.h>
+#include <aspect/global.h>
+
+#include <deal.II/base/quadrature_lib.h>
+#include <deal.II/fe/fe_values.h>
+
+#include <boost/archive/text_oarchive.hpp>
+#include <boost/archive/text_iarchive.hpp>
+
+namespace aspect
+{
+  namespace Postprocess
+  {
+    template <int dim>
+    std::pair<std::string,std::string>
+    DripDepth<dim>::execute (TableHandler &statistics)
+    {
+      const GeometryModel::Box<dim> *geometry
+        = dynamic_cast<const GeometryModel::Box<dim>*> (&this->get_geometry_model());
```

Comment:
Please change this to use the `Plugins::get_plugin_as_type` functions to avoid using `dynamic_cast`. Example from `benchmarks/annulus/annulus.cc`:
```
      const AnnulusMaterial<2> &
      material_model
        = Plugins::get_plugin_as_type<const AnnulusMaterial<2> >(this->get_material_model());
```